shell/lib/bootstrap.sh

```diff
@@ -190,7 +190,7 @@ stencil_arg() {
 # from stencil.lock.
 stencil_module_version() {
   local module_name="$1"
-  "$YQ" --raw-output ".modules | select(.name == \"$module_name\").version" "$(get_stencil_lock)"
+  "$YQ" --raw-output ".modules[] | select(.name == \"$module_name\").version" "$(get_stencil_lock)"
```

Comment:
So did this just never work before?

Reply:
Correct. Fortunately it currently is only triggered by the CircleCI orb sync script, which has limited use at the moment.